api/src/pcapi/core/offerers/synchronize_venues_banners_with_google_places.py

```diff
@@ -144,7 +145,15 @@ def synchronize_venues_banners_with_google_places(
     images_ignored_due_to_ratio = []
     banner_synchronized_venue_ids = set()
     for venue in venues_without_photos:
-        if not venue.googlePlacesInfo:
+        if venue.googlePlacesInfo:
+            if (
+                datetime.datetime.utcnow().month
+                - venue.googlePlacesInfo.updateDate.month
+                + (datetime.datetime.utcnow().year - venue.googlePlacesInfo.updateDate.year) * 12
+                > 2
+            ):
```

Comment:
Si je comprends bien, le but est de regarder si `updateDate` date de plus de 2 mois ? On devrait pouvoir l'écrire comme suit : 

```suggestion
            # Give up syncing after 2 months because [bla bla bla]
            if (venue.googlePlacesInfo.updateDate - datetime.datetime.utcnow()).days > 62:
```

Ça me semble plus clair et une assez bonne approximation.